content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,76 @@
+**Improved** the capability of the Sidecar API to customize `OutboundTrafficPolicy`.
+
+## Security
+
+**Added** a unified logging pattern to Citadel agent.
+**Added** metrics into Citadel agent to monitor SDS push.
+**Added** the capability to ingress gateway to provide client CA certificates when using SDS.
+**Added** the capability for trust domain validation when sidecars use mutual TLS.
+**Added** the labels `ca.istio.io/env` and `ca.istio.io/override`  to allow operators to control service account secret generation by namespace.
+**Removed** integration with Vault CA temporarily. This was caused by SDS requirements and will be reintroduced in a new release.
+**Improved** the performance of secrets handling in the sidecars by introducing SDS.
+**Improved** security and reliability by enabling the the Envoy JWT filter by default.
+**Improved** the reliability of SDS by requiring Kubernetes 1.13. Please be aware that while Istio supports Kubernetes 1.12, SDS [requires Kubernetes 1.13](/blog/2019/trustworthy-jwt-sds).
+
+## Telemetry
+
+**Added** support for Envoy gRPC Access Log Service [ALS](https://www.envoyproxy.io/docs/envoy/latest/api-v2/service/accesslog/v2/als.proto#grpc-access-log-service-als).
+**Added** a Grafana dashboard for Citadel monitoring.
+**Added** [metrics](/reference/commands/sidecar-injector/#metrics) for monitoring the sidecar injector webhook.
+**Added** control plane metrics to monitor Istio configuration state.
+**Added** telemetry reporting for traffic destined to `Passthrough` and `BlackHole` cluster.
+**Added** alpha support for in-proxy generation of service metrics using Prometheus.
+**Added** alpha support for environmental metadata in Envoy node metadata.
+**Added** alpha support for Proxy Metadata Exchange.
+**Added** alpha support for OpenCensus trace driver.
+**Improved** the mesh dashboard to provide monitoring of Istio configuration state.
+**Improved** the Pilot dashboard to expose additional key metrics to more clearly identify errors.
+**Removed** deprecated Adapter and Template custom resource definitions.
+**Deprecated** Mixer adapters. Extensions are expected to begin transition to new extension model. Legacy Mixer integration support will be maintained for 1.3 and 1.4.
+**Deprecated** Deprecated HTTP API Spec. The support for producing API attributes will be be removed in Istio 1.4.
+
+## Policy
+
+**Improved** rate limit enforcement to allow communication when the quota backend is unavailable.
+
+## Configuration Management
+
+**Added** the ability to configure Galley to use newer configuration processing.
+**Fixed** an issue in Galley where too many gRPC pings close connections.
+**Improved** Galley to avoid control plane upgrade failures.
+
+
+## istioctl
+
+**Added** many new istioctl experimental subcommands.
+- istioctl kube-uninject -f <descriptor.yaml> The inverse of kube-inject; removes Istio configuration from a Deployment descriptor.
```

Comment:
istioctl kube-uninject -> `istioctl kube-uninjected`, and for all others as well

Reply:
I think I have this one captured now.  please unresolve if not.